server/config/validation/paper-reply.js

```diff
@@ -335,13 +336,22 @@
       return null;
     }
 
-    if (!phoneRegex.test(value)) {
+    if (value.slice(0, 2) !== '44' && value.slice(0, 3) !== '+44' && value.slice(0, 4) !== '0044'
+    && value.slice(0, 2) !== '07' && value.slice(0, 2) !== '02') {
```

Comment:
02 (probably?) isn't the only initial digits in a phone number, but I'm unclear on which ones are likely to be in use.

Reply:
Addressed - now checking for different area codes.